tests/zmq/test_gnmi_zmq.py

```diff
@@ -9,6 +9,7 @@
 
 
 pytestmark = [
+    pytest.mark.disable_loganalyzer,
```

Comment:
Just suggestion, if error log is generated by config reload, then we could disable loganalyzer when reloading:
```
    loganalyzer[duthost.hostname].add_start_ignore_mark()
    config_reload()
    loganalyzer[duthost.hostname].add_end_ignore_mark()
```

Reply:
This will not work, because many service not fully start after config_reload command finish, it will take some time, on some slow planform for example Mellanox 2700, it will take few minutes.